src/components/NoteForm.js

```diff
@@ -35,6 +35,10 @@ function NoteForm({goalId, repoName, note}) {
     setEditing(!editing);
   };
 
+  const _handleCancelEditing = () => {
+    setEditing(false);
+  };
+
```

Comment:
This is redundant. `_handleToggleEditing` already does this, by flipping the true to false and vice versa.